src/plugin/markers/index.js

```diff
@@ -277,6 +279,13 @@ export default class MarkersPlugin {
             this.wavesurfer.fireEvent("marker-click", marker, e);
         });
 
+        labelDiv.addEventListener("contextmenu", (e) => {
```

Comment:
when using `addEventListener`, make sure to `removeEventListener` as well (e.g. in `destroy()` for example).

Reply:
Seems like the rest of the codebase assumes that removing an element from the DOM destroys its event listeners. The only event listeners currently being removed are ones on the document/body. If this is an oversight from earlier, let me know and I will also remove event listeners for mousedown, click etc.

Reply:
>  If this is an oversight from earlier, let me know and I will also remove event listeners for mousedown, click etc.

try creating and `destroy()`-ing the plugin a few times and see if the event listeners get triggered multiple times and increase. I'm almost certain this would happen..